.github/workflows/commit.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "CI"
 
-on:
+on: # yamllint disable-line rule:truthy
```

Comment:
This was throwing false positives my yamllint. https://github.com/adrienverge/yamllint/issues/430